lightning_client.go

```diff
@@ -1436,6 +1442,45 @@ func (s *lightningClient) ChannelBackups(ctx context.Context) ([]byte, error) {
 	return resp.MultiChanBackup.MultiChanBackup, nil
 }
 
+// SubscribeChannelBackups allows a client to sub-subscribe to the
```

Comment:
nit: `sub-subscribe` -> `subscribe`?

Reply:
ups it doesn't make sense.. I thing...... I copy/paste that part from `lnd/lnrpc/rpc.proto` 🤷‍♂️